tests/test_inject.py

```diff
@@ -1,12 +1,36 @@
+import logging
+import re
+import textwrap
+
 import pytest  # type: ignore
 
 from helpers import PIPX_METADATA_LEGACY_VERSIONS, mock_legacy_venv, run_pipx_cli, skip_if_windows
 from package_info import PKG
 
 
-def test_inject_simple(pipx_temp_env, capsys):
+@pytest.mark.parametrize(
+    "pkg_spec,",
+    [
+        PKG["black"]["spec"],  # was test_inject_simple
+        "jaraco.clipboard==2.0.1",  # was test_inject_tricky_character
+        "pylint==3.0.4",  # was test_spec
```

Comment:
This duplicates the `PKG["pylint"]["spec"]` test, but I wanted to explicitly match the previous tests. However I think this one could just be deleted, and the old test names removed.

Reply:
Yes, I agree.